Components/Text/Text.js

```diff
@@ -0,0 +1,55 @@
+import styled, { css } from 'styled-components';
+import { getColorValue } from '../Colors';
+
+
+const baseStyles = css`
+  -moz-osx-font-smoothing: grayscale;
+  -webkit-font-smoothing: antialiased;
+  font-style: normal;
+  color: ${getColorValue('black')};
+`;
+
+export const H1 = styled.h1`
+  ${baseStyles}
+  font-family: 'Karla', sans-serif;
```

Comment:
is declaring this font-family even necessary when i have it as the default body font?

Reply:
tested. not necessary.